.codeclimate.yml

```diff
@@ -0,0 +1,29 @@
+prepare:
+  fetch:
+  - url: "https://raw.githubusercontent.com/bespokepost/code-climate-config/master/config/rubocop.yml"
+    path: "config/rubocop.yml"
+engines:
+  rubocop:
+    enabled: true
+    config: config/rubocop.yml
+  brakeman:
+    enabled: true
+  csslint:
+    enabled: true
```

Comment:
@lucaslevin1 now that you're getting rid of activeadmin (which includes the only css files this project has) you will want to switch this to false after pulling these changes.

Reply:
Yessir